modules/globebrowsing/tile/tileprovider/presentationslideprovider.cpp

```diff
@@ -22,6 +22,8 @@
  * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
  ****************************************************************************************/
 
+#if 0
```

Comment:
1. We weren't planning on adding the presentation tile provider anyway but somehow it ended up in the master branch.
2. It does not work any more after the changes on how tiles are cached and reused and I did not  want to spend time on it.

Do you think I should remove it completely or try to fix it?